docs/INSTALLATION.md

```diff
@@ -2,7 +2,7 @@
 
 ## Prerequisites
 
-- Install [Node](https://nodejs.org/en/) using `nvm install` in each component folder.
+- Install [Node LTS Dubnium](https://nodejs.org/en/) or newer using `nvm install` in each component folder.
```

Comment:
You don't need to specify the version here as that is specified in the `nvmrc` file and if we change it there we will have to change it here.

Reply:
This has been resolved now.